src/main/java/land/oras/exception/OrasException.java

```diff
@@ -16,7 +16,7 @@ public class OrasException extends RuntimeException {
     /**
      * Logger
      */
-    private static final Logger LOG = LoggerFactory.getLogger(OrasException.class);
+    private static final Logger LOG = LoggerFactory.getLogger(land.oras.exception.OrasException.class);
```

Comment:
Why when it's imported already ? I would use only explicit class when class name colide. Otherwhise use import

Reply:
Done @jonesbusy